step-capstone/src/components/Pages/Trip.js

```diff
@@ -47,11 +47,11 @@ export default class Trip extends React.Component {
             },
             map: null,
             service: null,
-
             queryResults: null,
             placeIds: new Set(),
             showSuggestions: false,
-            selectedTimeslot: null
+            selectedTimeslot: null,
+            hotelMap: new Map()
```

Comment:
Nit: Since you have literal maps in this app, I find the name `hotelMap` a bit confusing. Maybe `hotelEventMap` or `hotelDateMap` would be clearer?

Reply:
Done!